array-methods/map.js

```diff
@@ -0,0 +1,14 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+const products = prices.map((price) => price / 2);
+
+const reformat = prices.map(elem => (
+  {
+    price: elem,
+    salePrice: elem / 2
+  }
+));
+console.log('Price object:', reformat);
+
+const roundedPrices = prices.map((e) => e.toFixed(2));
+const priceFormated = roundedPrices.map((e) => '$' + e.toString());
```

Comment:
Just as a tip, you could also do something like this:
``const priceFormated = prices.map((price) => `$${price.toFixed(2)}`);``